src/zope/tales/testing.py

```diff
@@ -0,0 +1,12 @@
+import six
+import unittest
+
+
+class TestCase(unittest.TestCase):
```

Comment:
I would consider `zope.tales.testing` to be public API (especially since there's a docstring and precedent in other zope packages). Is that really necessary? Do we want to support `zope.tales.testing.TestCase` "forever"? If not (I hope not), then perhaps this can go into to `zope.tales.tests`.

Reply:
Okay, I will move the class to zope/tales/tests/__init__.py it should only be necessary as long as we support Python 2.